OpenEdX/Router.swift

```diff
@@ -58,12 +58,13 @@ public class Router: AuthorizationRouter,
         navigationController.setViewControllers(viewControllers, animated: true)
     }
     
-    public func showMainOrWhatsNewScreen() {
+    public func showMainOrWhatsNewScreen(sourceScreen: LogistrationSourceScreen) {
         showToolBar()
         var storage = Container.shared.resolve(WhatsNewStorage.self)!
         let config = Container.shared.resolve(ConfigProtocol.self)!
 
         let viewModel = WhatsNewViewModel(storage: storage)
+        viewModel.sourceScreen = sourceScreen
```

Comment:
It seems we can pass this parameter as an argument to the ViewModel initializer.